common/util.go

```diff
@@ -184,6 +190,14 @@ func GetServiceClientForLocation(loc Location,
 		return ret, nil
 
 	case ELocation.File():
+		fileURLParts, err := file.ParseURL(resourceURL)
+		if err != nil {
+			return nil, err
+		}
+		fileURLParts.ShareName = ""
```

Comment:
Why we want to explicitly set these variables, if parseURL is not populating them then it shall be empty string already.

Reply:
We don't know if the URL passed is a service level URL, so we are forcing it here.